routes/api/job.js

```diff
@@ -1,5 +1,27 @@
-const router = require("express").Router();
+// const router = require("express").Router();
```

Comment:
Do your best to keep comments out of commits like this. Becomes stale very quickly.